sdk/search/search-documents/package.json

```diff
@@ -19,7 +19,7 @@
     "extract-api": "tsc -p . && api-extractor run --local",
     "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
     "integration-test:browser": "karma start --single-run",
-    "integration-test:node": "nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 5000000 --full-trace dist-esm/test/**/*.spec.js --harmony",
+    "integration-test:node": "nyc mocha --sort -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 5000000 --full-trace dist-esm/test/**/*.spec.js --harmony",
```

Comment:
Sorts the test files?

Reply:
:-) I am testing if adding this flag will force mocha to run tests only in sequence (non-parallel) But, PR checks are running only the Unit tests and not the integration tests :-(

Reply:
Just now triggered Integration tests manually..